### PR TITLE
gitignore: add ctags index file and gdb initialization script

### PR DESCRIPTION
**Pull request by jfischer-phytec-iot**
This PR adds ctags index and gdb initialization script files to gitignore.

### CODE REVIEW

**Comment by LudwigKnuepfer** [comment]
ACK

File: .gitignore [hunk 1]
```diff
@@ -24,6 +24,10 @@ cachegrind.out*
 *.kdev4
 # Codelite (among others) project files
 *.project
+# ctags index files
+tags
+# GDB initialization scripts
+.gdbinit
```
Comment (authmillenon) [inline .gitignore]: Is there a reason you need to put this script into the RIOT repository? Why not into your home directory?

Comment (jfischer-phytec-iot) [inline .gitignore]: For debugging session I like to make a local .gdbinit, similar to:
```
file bin/pba-d-01-kw2x/usbtest.elf

dir .
set prompt (cm4-gdb) 
set print pretty

# connect to openOCD running on gdb port 3333
target remote localhost:3333

set arm force-mode thumb
```
and start openocd with ```make debug-server```. So I can use cgdb as a gdb frontend and incidentally also see logs of openocd.

**Comment by OlegHahm** [comment]
ACK

File: .gitignore (hunk 1 above)
Comment (authmillenon) [inline .gitignore]: Okay. Just was curious :-). ACK then.

**Comment by cgundogan** [comment]
and GO

**Pull request merged**